pkg/init/cmd/service/runc.go

```diff
@@ -50,42 +51,78 @@ func runcInit(rootPath string) int {
 			status = 1
 			continue
 		}
+
 		pidfile := filepath.Join(tmpdir, name)
 		cmd := exec.Command(runcBinary, "create", "--bundle", path, "--pid-file", pidfile, name)
 
 		stdoutFile := filepath.Join("/var/log", name+".out.log")
-		stderrFile := filepath.Join("/var/log", name+".err.log")
 
-		stdoutFd, err := os.OpenFile(stdoutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+		stdoutFd, stdoutFdErr := os.OpenFile(stdoutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
 		defer stdoutFd.Close()
-		if err == nil {
-			cmd.Stdout = stdoutFd
+
+		var stdoutCopyErr error
+
+		if stdoutFdErr == nil {
+			stdoutIn, _ := cmd.StdoutPipe()
+			stdoutWriter := io.MultiWriter(os.Stdout, stdoutFd)
+			go func() {
+				_, stdoutCopyErr = io.Copy(stdoutWriter, stdoutIn)
+			}()
 		} else {
+			log.Printf("Could not open %s for writing: %v", stdoutFile, stdoutFdErr)
 			cmd.Stdout = os.Stdout
-			log.Printf("Could not open %s for writing", stdoutFile)
 		}
 
-		stderrFd, err := os.OpenFile(stderrFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+		stderrFile := filepath.Join("/var/log", name+".err.log")
+
+		stderrFd, stderrFdErr := os.OpenFile(stderrFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
 		defer stderrFd.Close()
-		if err == nil {
-			cmd.Stderr = stderrFd
+
+		var stderrCopyErr error
+
+		if stderrFdErr == nil {
+			stderrIn, _ := cmd.StderrPipe()
+			stderrWriter := io.MultiWriter(os.Stderr, stderrFd)
+			go func() {
+				_, stderrCopyErr = io.Copy(stderrWriter, stderrIn)
+			}()
```

Comment:
if a stderr log file can be opened for write/append, then io.Copy the cmd.StderrPipe to a io.MultiWriter (of both the os.Stderr and the stderr log file)